src/plugins/unified_search/public/query_string_input/esql_menu_popover.tsx

```diff
@@ -65,32 +82,44 @@ export const ESQLMenuPopover = () => {
       </EuiContextMenuItem>
     );
     return panelItems;
-  }, [docLinks.links.query.queryESQL, docLinks.links.query.queryESQLExamples]);
+  }, [
+    docLinks.links.query.queryESQL,
+    docLinks.links.query.queryESQLExamples,
+    toggleLanguageComponent,
+  ]);
 
   return (
-    <EuiPopover
-      button={
-        <EuiButton
-          color="text"
-          onClick={() => setIsESQLMenuPopoverOpen(!isESQLMenuPopoverOpen)}
-          data-test-subj="esql-menu-button"
-          size="s"
-        >
-          {i18n.translate('unifiedSearch.query.queryBar.esqlMenu.label', {
-            defaultMessage: 'ES|QL help',
-          })}
-        </EuiButton>
-      }
-      panelProps={{
-        ['data-test-subj']: 'esql-menu-popover',
-        css: { width: 240 },
-      }}
-      isOpen={isESQLMenuPopoverOpen}
-      closePopover={() => setIsESQLMenuPopoverOpen(false)}
-      panelPaddingSize="s"
-      display="block"
-    >
-      <EuiContextMenuPanel size="s" items={esqlPanelItems} />
-    </EuiPopover>
+    <>
+      <EuiPopover
+        button={
+          <EuiButton
+            color="text"
+            onClick={() => setIsESQLMenuPopoverOpen(!isESQLMenuPopoverOpen)}
+            data-test-subj="esql-menu-button"
+            size="s"
+          >
+            {i18n.translate('unifiedSearch.query.queryBar.esqlMenu.label', {
+              defaultMessage: 'ES|QL help',
+            })}
+          </EuiButton>
+        }
+        panelProps={{
+          ['data-test-subj']: 'esql-menu-popover',
+          css: { width: 240 },
+        }}
+        isOpen={isESQLMenuPopoverOpen}
+        closePopover={() => setIsESQLMenuPopoverOpen(false)}
+        panelPaddingSize="s"
+        display="block"
+      >
+        <EuiContextMenuPanel size="s" items={esqlPanelItems} />
+      </EuiPopover>
+      <LanguageDocumentationFlyout
```

Comment:
<img width="848" alt="Screenshot 2024-09-13 at 11 00 10" src="https://github.com/user-attachments/assets/09ba6fe4-b779-4dc7-a064-37df26de6a15">
I believe you are aware of this, but you can drag the resizable flyout over the accordion item at the bottom

Reply:
This is not caused my PR but maybe my follow up PR accidentally also fixes it https://github.com/stratoula/kibana/pull/33. Definitely irrelevant with the scope of this PR

Reply:
here it is https://github.com/elastic/kibana/issues/192876

Reply:
Marco created an issue here https://github.com/elastic/kibana/issues/192876